ecl/hqlcpp/hqliproj.ipp

```diff
@@ -158,6 +158,7 @@ public:
 protected:
     bool contains(IHqlExpression & field) const;
     bool contains(IAtom * name) const; // debugging only
+    IHqlExpression * findByName(IAtom * name) const; // debugging only
```

Comment:
'debugging only'? I looked through a level or two up but couldn't find anything that looked like conditional code to suggest that this was called in a debug context only. I guess this refers to something else?